.editorconfig

```diff
@@ -100,6 +100,7 @@ csharp_style_expression_bodied_methods = true:suggestion
 csharp_style_expression_bodied_operators = true:suggestion
 csharp_style_expression_bodied_properties = true:suggestion
 csharp_style_inlined_variable_declaration = true:suggestion
+csharp_style_namespace_declarations = block_scoped:warning
```

Comment:
Just a final nitpick, but I think that the importance should be set to `suggestion` instead of `warning` to match the rest of the style rules defined :D